internal/idler/user_idler.go

```diff
@@ -99,12 +103,14 @@ func (idler *UserIdler) Run(wg *sync.WaitGroup, ctx context.Context, cancel cont
 				return
 			case idler.user = <-idler.userChan:
 				idler.logger.WithField("data", idler.user.String()).Info("Received user data.")
+
 				err := idler.checkIdle()
 				if err != nil {
 					idler.logger.WithField("error", err.Error()).Warn("Error during idle check.")
 				}
 			case <-ticker:
```

Comment:
> And do we want to use timeout rather than ticker again as noted in #127 ?

I think we might want both. A timer for doing the idle check based on timeout after the last received event and a ticker which ensures that the counter gets reset. 

Either way, I'd prefer to address this separately in #127

Reply:
> Should there be a checkIdle rather than idleAfter

I guess the naming of the variable is just bad. Let me fix that.

Reply:
pushed update